js/mercado.js

```diff
@@ -350,10 +395,55 @@ module.exports = class mercado extends Exchange {
         };
     }
 
+    parseOHLCV (ohlcv, market = undefined, timeframe = '1m', since = undefined, limit = undefined) {
+        return [
+            ohlcv['timestamp'] * 1000,
+            parseFloat (ohlcv['open']),
+            parseFloat (ohlcv['high']),
+            parseFloat (ohlcv['low']),
+            parseFloat (ohlcv['close']),
+            parseFloat (ohlcv['volume']),
+        ];
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = this.market (symbol);
+        let request = {
+            'precision': this.timeframes[timeframe],
+            'coin': market.id.toLowerCase (),
+        };
+        if (since !== undefined) {
+            request['from'] = parseInt (since / 1000);
+            if (limit !== undefined) {
+                request['to'] = (this.sum (request['from'], limit * this.parseTimeframe (timeframe)));
```

Comment:
This set of conditionals will not work if `limit` is defined and `since` is undefined... If `since` is undefined, `limit` is ignored which has to be fixed.